galaxy.yml

```diff
@@ -16,8 +16,7 @@ description: Install and configure RHTAS, a downstream redistribution of the Sig
 license_file: Apache-2.0
 tags: [sigstore, tas, rhtas, security, cosign]
 # NOTE: when updating, also update dependencies in requirements.yml
-dependencies:
-  containers.podman: ">=1.15.0"
+dependencies: {}
 repository: https://github.com/securesign/artifact-signer-ansible/
 documentation: http://TODO.com
```

Comment:
Are we wanting to add the official product documentation link here?

Reply:
Oh yeah, I totally forgot about this. Thanks for noticing, I'll open a separate PR, I don't want to mix it in here.